src/controllers/userController/sendEmail.js

```diff
@@ -0,0 +1,27 @@
+import nodemailer from 'nodemailer';
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+const transporter = nodemailer.createTransport({
+  host: 'smtp.gmail.com',
+  port: 465,
+  secure: true, // use SSL
+  auth: {
+    user: process.env.GMAIL_EMAIL,
+    pass: process.env.GMAIL_PASSWORD
+  }
+});
+
+const sendVerificationEmail = async (data) => {
```

Comment:
you can put this in helper forlder o util forlder as peaple are going to use it not only this controller